docs/trace_collection.md

```diff
@@ -279,55 +533,156 @@ To provide a continuous trace inside a RxJava stream you need to follow the step
    ```
 2. Then in your project open a scope when the Observable is subscribed and close it when it completes. Any span
    created inside the stream operators will be displayed inside this scope (parent Span):
-   ```kotlin
-   Single.fromSupplier{}
+
+{{< tabs >}}
+{{% tab "Kotlin" %}}
+```kotlin
+    Single.fromSupplier{}
         .subscribeOn(Schedulers.io())
-        .map { 
+        .map {
             val span = GlobalTracer.get().buildSpan("<YOUR_OP_NAME>").start()
             // ...
             span.finish()
         }
-       .doOnSubscribe {
-           val span = GlobalTracer.get()
-               .buildSpan("<YOUR_OP_NAME>")
-               .start()
-           GlobalTracer.get().scopeManager().activate(span)
-       }
-       .doFinally {
-           GlobalTracer.get().scopeManager().activeSpan()?.let {
-               it.finish()
-           }
-       }
-   ```
+        .doOnSubscribe {
+            val span = GlobalTracer.get()
+                    .buildSpan("<YOUR_OP_NAME>")
+                    .start()
+            GlobalTracer.get().scopeManager().activate(span)
+        }
+        .doFinally {
+            GlobalTracer.get().scopeManager().activeSpan()?.let {
+                it.finish()
+            }
+        }
+```
+{{% /tab %}}
+{{% tab "Java" %}}
+```java
+    Single.fromSupplier(new Supplier<Object>() {
+                    @Override
+                    public Object get() throws Throwable {
+                        return null;
```

Comment:
RxJava 2 and 3 don't allow `null` values in the stream. This example doesn't work I think (and I guess same for the Kotlin example).

Maybe use `new Object()` or just write `...`?